lucien-ios/AppDelegate.swift

```diff
@@ -19,7 +19,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         self.window = window
         window.rootViewController = RootViewController()
         window.makeKeyAndVisible()
+
+//        let navController = UINavigationController()
```

Comment:
Commented-out code should not be in the final pull request - it's okay to leave it in a commit so you can get it back later, but it should be removed before you put the PR up